addon/components/old-file-browser/style.scss

```diff
@@ -0,0 +1,34 @@
+.file-browser {
```

Comment:
Can we change this to old-file-browser to ensure no collision with new file-browser?

Reply:
right!